apps/web/lib/api/links/utils/transform-link.ts

```diff
@@ -9,25 +8,11 @@ export type LinkWithTags = Link & {
 export const transformLink = (link: LinkWithTags) => {
   const tags = (link.tags || []).map(({ tag }) => tag);
 
-  const shortLink = linkConstructor({
-    domain: link.domain,
-    key: link.key,
-  });
-
-  const qrLink = linkConstructor({
-    domain: link.domain,
-    key: link.key,
-    searchParams: {
-      qr: "1",
-    },
-  });
-
   return {
     ...link,
-    shortLink,
     tagId: tags?.[0]?.id ?? null, // backwards compatibility
     tags,
-    qrCode: `https://api.dub.co/qr?url=${qrLink}`,
+    qrCode: `https://api.dub.co/qr?url=${link.shortLink}`,
```

Comment:
`qrLink` needs `?qr=1` to identify the scan from the QR code.

Reply:
Sure I'll add this param